quickstart-mongodbatlas/README.md

```diff
@@ -0,0 +1,28 @@
+quickstart-mongodbatlas
+=============
+## MongoDB Atlas on the AWS Cloud
+
+Start here: [beta-launch.md](beta-launch.md)
+
+This Quick Start sets up a flexible, scalable AWS environment for MongoDB Atlas through AWS CloudFormation resources.
+
+MongoDB Atlas is the global cloud database service for modern applications. Deploy fully managed MongoDB across AWS, Azure, or GCP. Best-in-class automation and proven practices guarantee availability, scalability, and compliance with the most demanding data security and privacy standards.
+
+The Quick Start offers two deployment options:
+
+- Deploying MongoDB Atlas into a new virtual private cloud (VPC) on AWS
+- Deploying MongoDB Atlas into an existing VPC on AWS
```

Comment:
Missed this last time but probably should just remove as an artifact of the current one.  We can deploy Atlas on an existing VPC they have (unless it's already in Atlas but that's slightly different).